screens/auth/LoginScreen.js

```diff
@@ -107,25 +127,81 @@ export default class Login extends React.Component {
       .catch(err => console.error(err));
   }
 
+  onFocus(index) {
+    this.setState({
+      [this.state.booleans[index]]: true
+    });
+  }
+
+  onBlur(index) {
+    this.setState({
+      [this.state.booleans[index]]: false
+    });
+  }
+
   render() {
     return (
-      <LoginContainer>
-        <Input
-          placeholder="Phone Number (i.e. 1234567890)"
-          keyboardType="number-pad"
-          maxLength={10}
-          value={this.state.phoneNumber}
-          onChangeText={text => this.setState({ phoneNumber: text })}
-        />
-        <Input
-          placeholder="Password"
-          secureTextEntry
-          onChangeText={text => this.setState({ password: text })}
-          value={this.state.password}
-        />
-        <Button title="Log In" onPress={() => this.handleSubmit()} />
+      <AuthScreenContainer>
+        <BigTitle>Log in</BigTitle>
+        <FormContainer>
+          <TextFieldContainer>
+            <Caption
+              color={
+                this.state.phoneNumBool
+                  ? Colors.primaryGreen
+                  : Colors.activeText
+              }>
+              {' '}
+              Phone Number
+            </Caption>
+            <TextField
+              onBlur={() => this.onBlur(0)}
+              onFocus={() => this.onFocus(0)}
+              placeholder="Phone Number"
+              keyboardType="number-pad"
+              maxLength={10}
+              value={this.state.phoneNumber}
+              onChangeText={text => this.setState({ phoneNumber: text })}
+              borderColor={
+                this.state.phoneNumBool
+                  ? Colors.primaryGreen
+                  : Colors.activeText
+              }
+            />
+          </TextFieldContainer>
+
+          <TextFieldContainer>
```

Comment:
Mistake in Figma, but this should have the top caption as well like the other fields.